pkgs/os-specific/darwin/linearmouse/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenvNoCC, fetchurl }:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "linearmouse";
+  version = "0.9.5";
+
+  src = fetchurl {
+    url = "https://github.com/linearmouse/linearmouse/releases/download/v${finalAttrs.version}/LinearMouse.dmg";
+    hash = "sha256-T25H2CmDBt3J5zaC8gy87PyJONPsDZpF7S5eksRzgrA=";
+  };
+
+  sourceRoot = "LinearMouse.app";
+
+  # https://discourse.nixos.org/t/help-with-error-only-hfs-file-systems-are-supported-on-ventura/25873
+  unpackCmd = ''
```

Comment:
What are you trying to do here?

Reply:
`undmg` doesn't support unpacking APFS dmgs, so we use some of the macOS built-ins to do that for us. I'll leave a comment further explaining the issue.